src/ahrens_lab_to_nwb/yu_mu_cell_2019/yu_mu_cell_2019_metadata.yml

```diff
@@ -1,11 +1,71 @@
 NWBFile:
   related_publications: https://doi.org/10.1016/j.cell.2019.05.050
   session_description: |
-    When a behavior repeatedly fails to achieve its goal, animals often give up and become passive, which can be strategic for preserving energy or regrouping between attempts. It is unknown how the brain identifies behavioral failures and mediates this behavioral-state switch. In larval zebrafish swimming in virtual reality, visual feedback can be withheld so that swim attempts fail to trigger expected visual flow. After tens of seconds of such motor futility, animals became passive for similar durations. Whole-brain calcium imaging revealed noradrenergic neurons that responded specifically to failed swim attempts and radial astrocytes whose calcium levels accumulated with increasing numbers of failed attempts. Using cell ablation and optogenetic or chemogenetic activation, we found that noradrenergic neurons progressively activated brainstem radial astrocytes, which then suppressed swimming. Thus, radial astrocytes perform a computation critical for behavior: they accumulate evidence that current actions are ineffective and consequently drive changes in behavioral states.
+    When a behavior repeatedly fails to achieve its goal, animals often give up and become passive, which can be strategic for preserving energy or regrouping between attempts. It is unknown how the brain identifies behavioral failures and mediates this behavioral-state switch. In larval zebrafish swimming in virtual reality, visual feedback can be withheld so that swim attempts fail to trigger expected visual flow. After tens of seconds of such motor futility, animals became passive for similar durations. Whole-brain calcium imaging revealed noradrenergic neurons that responded specifically to failed swim attempts and radial astrocytes whose calcium levels accumulated with increasing numbers of failed attempts. Using cell ablation and optogenetic or chemogenetic activation, we found that noradrenergic neurons progressively activated brainstem radial astrocytes, which then suppressed swimming. Thus, radial astrocytes perform a computation critical for behavior; they accumulate evidence that current actions are ineffective and consequently drive changes in behavioral states.
   institution: Janelia Research Campus
-  lab: Misha Ahrens
+  lab: Ahrens Lab
   experimenter:
     - Mu, Yu
-    - Ahrens, Misha
+    - Bennett, Davis V.
+    - Rubinov, Mikail
+    - Narayan, Sujatha
+    - Yang, Chao-Tsung
+    - Tanimoto, Masashi
+    - Mensh, Brett D.
+    - Looger, Loren L.
+    - Ahrens, Misha B.
 Subject:
   species: Danio rerio
+
+Ophys:
+  Device:
+    - name: Microscope
+      description: Confocal imaging was done using a Zeiss LSM 710, LSM 880 upright confocal, or a Zeiss LSM 800.
```

Comment:
You could add manufacturer as well here to pynwb.Device:
https://pynwb.readthedocs.io/en/stable/pynwb.device.html

Reply:
Will do, at least we know that manufacturer in all cases